### PR TITLE
Added info about status/restart for the OVA services

### PR DESCRIPTION
**Pull request by stuclem**
Fixes https://github.com/vmware/vic-product/issues/135.

Very short topics on restarting the different services. It would be great if we could add some information about what might cause the services to fail or die, and how to remedy that. @frapposelli, any ideas?

Thanks!

### CODE REVIEW

File: docs/user_doc/vic_vsphere_admin/restart_services.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+You can restart the vSphere Integrated Containers Registry and vSphere Integrated Containers Management Portal services and the file server that run in the appliance by logging in to the vSphere Integrated Containers appliance.
+
+**Prerequisites**
+
+You deployed the vSphere Integrated Containers appliance.
+
+**Procedure**
+1. Connect to the vSphere Integrated Containers appliance by using SSH.
+2. If the status of the service is `active (running)`, run one of the following commands to restart one of the vSphere Integrated Containers services:
```
Comment (frapposelli) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: service doesn't have to be in state `active (running)`, a restart will try a restart even if the service is currently dead.

File: docs/user_doc/vic_vsphere_admin/restart_services.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  - vSphere Integrated Containers Registry: `systemctl restart harbor`
+  - vSphere Integrated Containers Management Portal services: `systemctl restart admiral`
+  - Embedded file server: `systemctl restart fileserver`
+2. If the status of the service is `failed` or `inactive (dead)`, run one of the following commands to restart one of the vSphere Integrated Containers services:
```
Comment (frapposelli) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: see above

File: docs/user_doc/vic_vsphere_admin/service_status.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+|Status|Description|
+|---|---|
+|`active (running)`|The service is running correctly.|
+|`inactive (failed)`|The service failed to start.|
+|`inactive (dead)`|The service is not responding.|
+
+**What to Do Next**
+
+If the status is `inactive`, see [Restart the vSphere Integrated Containers Services](restart_services.md).
```
Comment (frapposelli) [inline docs/user_doc/vic_vsphere_admin/service_status.md]: or `dead`.

File: docs/user_doc/vic_vsphere_admin/restart_services.md (hunk 1 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/restart_services.md (hunk 2 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/restart_services.md]: Fixed

File: docs/user_doc/vic_vsphere_admin/service_status.md (hunk 3 above)
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/service_status.md]: Fixed

**Comment by stuclem** [comment]
Thanks @frapposelli - I also just noticed that the email settings were removed from the OVA in PR https://github.com/vmware/vic/pull/4556, so I removed them too.

**Pull request merged**